mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -135,17 +139,20 @@ def __init__(
             parquet_path=self.parquet_path,
         )
 
-    def create_feature_set(self):
-        feature_set = fs.FeatureSet(
-            "monitoring", entities=[ENDPOINT_ID], timestamp_key=TIMESTAMP
-        )
-        feature_set.metadata.project = self.project
-        feature_set.graph.to(
+    def apply_serving_graph(self, fn):
+        """
+        Apply monitoring serving graph to a given serving function
+
+        :param fn: A serving function
+        """
+        graph = fn.set_topology("flow")
+        graph.to(
```

Comment:
we already talked about it in previous PR, this serving graph needs to contain documentation for each step, or even better add `apply_x` for each step which the name represents what is being done, because you are only passing `self` and `fn` this won't require passing to each function all the params that are being part of the step.